spec/fixtures/users.yml

```diff
@@ -1,11 +1,12 @@
-# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+# https://github.com/trkin/rails_minitest/blob/main/test/fixtures/users.yml
+DEFAULTS: &DEFAULTS
+  email: $LABEL@example.com
+  encrypted_password: <%= User.new.send(:password_digest, 'password') %>
+  confirmed_at: <%= Time.zone.now %>
 
-# This model initially had no columns defined. If you add columns to the
-# model remove the "{}" from the fixture names and add the columns immediately
-# below each fixture, per the syntax in the comments below
-#
-one:
-  email: one@example.com
+user:
+  <<: *DEFAULTS
 
-two:
-  email: two@example.com
+unconfirmed:
+  <<: *DEFAULTS
+  confirmed_at:
```

Comment:
Good job so far.
Please explain the source for this code.

Reply:
There is a link to the source at the top of file

https://github.com/trkin/rails_minitest/blob/main/test/fixtures/users.yml

@trkin/internship the idea is to create common knowledge so we can use those examples in various projects and improve them. I'm still thinking how to organise examples to be easy to understand and compatible with different use case.